project/settings.py

```
@@ -1,5 +1,6 @@
 from pathlib import Path
 from decouple import config
+import dj_database_url
 
 import dj_database_url
```

Ruff (0.8.2)

Redefinition of unused `dj_database_url` from line 3

Remove definition: `dj_database_url`

(F811)